### ISSUE TITLE
docs.dask.org gives me a CloudFlare 1014 error

### ISSUE THREAD

**Issue by jadonn** [dask/dask.github.io#45]
This is not a code bug error but a hosting/system error. If this is not the right place I am happy to re-file.

When I try to view the documents for Dask, I see a CloudFlare error. The error page says:

Error 1014 Ray ID: 61f761bdab3924ea • 2021-02-10 16:59:38 UTC
CNAME Cross-User Banned
What happened?
You've requested a page on a website that is part of the Cloudflare network. The host is configured as a CNAME across accounts on Cloudflare, which is prohibited by security policy.

What can I do?
If you are interested in learning more about Cloudflare, please visit our website.

Cloudflare Ray ID: 61f761bdab3924ea • Your IP: REDACTED • Performance & security by Cloudflare
--- End of Error ---

If it makes any difference, I'm in the eastern United States. I can provide a screenshot if necessary.

**Comment by quasiben** [comment]
Hmm, i just tried and didn't see an issue.  Can you try again ?

**Comment by jadonn** [comment]
Weird! I do not get the error when I browse the docs in Vivaldi for Android on my home internet, but I still get the error on my desktop (Fedora) device on my home internet. I get it using Firefox, Chrome, Vivaldi, and curl. I also get the error when I connect to a VPN from my desktop device and attempt to load the page.

However, another computer (Windows 10) I have does not have the error. I suppose it's just this one particular machine that for whatever reason makes Cloudflare refuse to load the web page. I'm not blaming the fact that this device runs Fedora/Linux, but if I had to guess it's probably something I've done to my installation of Fedora without realizing.

Thanks for checking this for me! I am sorry I did not check other devices first. I have never heard of Cloudflare disliking a single individual device like this so it did not occur to me this could be a client-side issue.
